x-pack/solutions/security/plugins/security_solution/common/endpoint/constants.ts

```diff
@@ -109,6 +109,9 @@ export const ACTION_STATE_ROUTE = `${BASE_ENDPOINT_ACTION_ROUTE}/state`;
 /** Endpoint Agent Routes */
 export const AGENT_STATUS_ROUTE = `/internal${BASE_ENDPOINT_ROUTE}/agent_status`;
 
+/** Custom Scripts Routes */
+export const CUSTOM_SCRIPTS_ROUTE = `/internal${BASE_ENDPOINT_ROUTE}/custom_scripts`;
```

Comment:
This API is part of response actions, but the route is defined under our base API path - maybe consider: `/internal/api/endpoint/action/custom_scripts`?

Reply:
Sure, I can change this 👍 But again -  just to confirm, are we treating this as a response action now? I was under the impression it wasn’t the case.